### PR TITLE
cam6_2_025?: Mods to resolve issue #130 SCAM6 namelist definitions inconsistent

### PR DESCRIPTION
**Pull request by jtruesdal**
The PR takes closes issue #130 where there were erroneous default values in the namelist definition file.  I also included some minor cleanup to the scam iop variables as well as corrected a bug for the upcoming SAS IOP file.

### CODE REVIEW

**Comment by jtruesdal** [comment]
I made a scam test on izumi with and without one of the problematical scam namelist variables and it built and ran without error.  The other mods only affect SCAM files or are within a if(single_column) statement.  Let me know what other testing you would like me to do.

**Comment by cacraigucar** [comment]
I added this to the Projects for "CAM development branch".  Does this bug also need to be fixed on the "Release branch"?  If so, please add to the "Projects" section, the "release" project.

File: src/control/scamMod.F90 [hunk 1]
```diff
@@ -236,7 +236,7 @@ subroutine scam_readnl(nlfile,single_column_in,scmlat_in,scmlon_in)
        scm_cambfb_mode,scm_crm_mode,scm_zadv_uv,scm_zadv_T,scm_zadv_q,&
        scm_use_obs_T, scm_use_obs_uv, scm_use_obs_qv, &
        scm_relax_linear, scm_relax_tau_top_sec, &
-       scm_relax_tau_bot_sec, scm_force_latlon, scm_relax_fincl
+       scm_relax_tau_bot_sec, scm_force_latlon, scm_relax_fincl, scm_backfill_iop_w_init
```
Comment (gold2718) [inline src/control/scamMod.F90:239]: I would like to see more of these values logged (i.e., written to the log file). I realize that this has not been happening for SCAM over the years but now is as good a time as ever to make up for lost time.

**Comment by nusbaume** [comment]
@cacraigucar It looks like issue #130 is already listed on the "release" project page.  So I don't think any new project info needs to be added until a PR is made to the release branch itself.  Unless of course I am missing something?

File: src/control/scamMod.F90 (hunk 1 above)
Comment (jtruesdal) [reply src/control/scamMod.F90:239]: Agreed.  I will update readnl to log the SCAM namelist parameters and take a look through the routines for other opportunities to refactor.

Comment (jtruesdal) [reply src/control/scamMod.F90:240]: @gold2718  SCAM namelist parameters now written to log file at end of readnl.

**Comment by nusbaume** [comment]
@cacraigucar I removed the public release project link, as a PR can only be pushed to one branch on the repo, which in this case is `cam_development`.  @jtruesdal will need to make a second PR in order to add these changes to the release branch.  I should note that this is a fundamental git/github limitation, so there isn't much we can do about it without putting in a serious coding effort.  In fact the issue-closing script will throw an error if more than one project is associated with a particular PR.  I forgot to add that detail to the wiki, so I will do so now.

Issues, on the other hand, can be associated with as many projects as we want, which is the case with issue #130, which this PR is attempting to fix for the development branch.  

Anyways, if you are confused by any of this just let me know.  Thanks!

**Pull request merged**